apps/www/registry/stories/badge.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import { Badge } from "@/registry/default/ui/badge"
+
+/**
+ * Displays a badge or a component that looks like a badge.
+ */
+const meta: Meta<typeof Badge> = {
+  title: "ui/Badge",
+  component: Badge,
+  tags: ["autodocs"],
+  argTypes: {},
+  parameters: {
+    layout: "centered",
+  },
+}
+export default meta
+
+type Story = StoryObj<typeof Badge>
+
+/**
+ * The default form of the badge.
+ */
+export const Default: Story = {
+  render: (args) => <Badge {...args}>Badge</Badge>,
+  args: {},
+}
```

Comment:
This would be more idiomatic CSF:

1. Move the "Badge" text to `args`:

   ```js
   export const Default: Story = {
     render: (args) => <Badge {...args} />,
     args: {
       children: 'Badge',
     },
   }
   ```

1. Now the `render` function simply passes all `args` along to the component, which is the same as the implicit default. Thus, it can be removed from the story definition.

1. Finally, because you're sharing the `render` across all stories, it would be most accurate to define the `children` arg value at the meta level.

Which leads to this full suggestion:

```suggestion
const meta: Meta<typeof Badge> = {
  title: "ui/Badge",
  component: Badge,
  tags: ["autodocs"],
  argTypes: {},
  args: {
    children: 'Badge',
  },
  parameters: {
    layout: "centered",
  },
}
export default meta

type Story = StoryObj<typeof Badge>

/**
 * The default form of the badge.
 */
export const Default: Story = {
  args: {},
}
```

You can then remove `render: Default.render` from the other stories.